scene_exceptions/scene_exceptions_tvdb.json

```diff
@@ -682,6 +682,13 @@
                 "Dragons Den UK"
             ]
         },
+        "79895": {
+            "8": [
+                "Gintama S4",
+                "Gintama.",
```

Comment:
Thanks for the PR!

Is this name really needed? ```Gintama.``` (with the ending dot). It shouldn't make a difference.